### ISSUE TITLE
codemetar gets confused by multiple git remotes

### ISSUE THREAD

**Issue by Bisaloo** [ropensci/codemetar#276]
I have a local fork of codemetar with two remotes:

```bash
git remote -v
```
> bisaloo git@github.com:Bisaloo/codemetar.git (fetch)
bisaloo git@github.com:Bisaloo/codemetar.git (push)
origin  git@github.com:ropensci/codemetar.git (fetch)
origin  git@github.com:ropensci/codemetar.git (push)

I believe this is a standard setup to submit PRs when you don't have write access to the upstream repo (`ropensci/codemetar` in this case).

If I run 

```r
devtools::load_all()
write_codemetar()
```

with this setup, I get the following diff 

```
@@ -537,7 +537,7 @@
   ],
   "isPartOf": "https://ropensci.org",
   "keywords": ["metadata", "codemeta", "ropensci", "citation", "credit", "linked-data", "json-ld", "r", "rstats", "r-package", "peer-reviewed"],
-  "releaseNotes": "https://github.com/ropensci/codemetar/blob/master/NEWS.md",
+  "releaseNotes": "https://github.com/Bisaloo/codemetar/blob/master/NEWS.md",
   "fileSize": "288.075KB",
   "contIntegration": ["https://travis-ci.org/ropensci/codemetar", "https://ci.appveyor.com/project/cboettig/codemetar/branch/master", "https://codecov.io/github/ropensci/codemetar?branch=master"],
   "developmentStatus": "http://www.repostatus.org/#active",
@@ -546,5 +546,5 @@
     "url": "https://github.com/ropensci/software-review/issues/130",
     "provider": "https://ropensci.org"
   },
-  "readme": "https://github.com/ropensci/codemetar/blob/master/README.md"
+  "readme": "https://github.com/Bisaloo/codemetar/blob/master/README.md"
 }
```

which suggests that codemetar is using the wrong remote (the first it encounters?) to create `codemeta.json`.

I'm not entirely sure how this could be solved just yet :thinking:. Maybe always use the repo labelled as `origin`?

**Comment by cboettig** [comment]
Thanks for the bug report and good question.  Yeah, using `origin` makes sense, (though technically I guess a user isn't required to have a remote named `origin` so a fall-back for that edge case would be good).  

We could force an interactive prompt in this case, but I think it's important that we maintain a good way to generate codemeta non-interactively as well.  

@maelle thoughts?

**Comment by maelle** [comment]
What about origin if it exists, and the first remote otherwise?

**Comment by maelle** [comment]
I've just forked and cloned the forked repo and have a single remote corresponding to the forked repo :thinking: 

I forked using the web interface, and cloned via ssh.

@Bisaloo how did you fork the repo and add the upstream URL?

**Comment by Bisaloo** [comment]
I always manually add the second remote.

Forking workflow:

1. Clone the upstream repo via ssh (only remote is `git@github.com:ropensci/codemetar.git`)
1. Fork the repo via GitHub web interface
1. Add the second remote with `git remote add bisaloo git@github.com:Bisaloo/codemetar.git`

Submitting a PR:

1. Create a new branch, make my changes and commit them
1. Push this new branch to my fork (`git push bisaloo`) as I don't have write access to the upstream repo
1. Create a PR via GitHub web interface

The addition of the second remote is required because of the 2nd step to submit a PR. Unless I am missing something and I've been making my life more complicated this whole time :laughing:

**Comment by maelle** [comment]
aaah I clone the fork directly! in that case origin is maelle/codemetar (and I simply write git push). But maybe my method is less easy for syncing?

**Comment by Bisaloo** [comment]
Yeah, I used to clone my fork as well but then I found it was a pain to stay in sync with upstream.

But you make me realize that some people might clone their fork and then add another (e.g. `upstream`) remote to stay in sync. In this case, the idea to use `origin` as the ""correct"" remote wouldn't work.

**Comment by maelle** [comment]
what I also realize is that codemetar should use the CodeRepository field, not guess GitHub, for release notes and the README. :grimacing:

**Comment by maelle** [comment]
@Bisaloo can you please try 

```r
remotes::install_github("ropensci/codemetar@fixreadmereleaseurl")
codemetar::write_codemeta()
```

and report here? Thanks in advance. :-)

**Comment by Bisaloo** [comment]
Tested just now and it works like a charm!

**Comment by maelle** [comment]
Thanks a lot, and great news! :tada:

**Comment by maelle** [comment]
fixed in #278
